pyproject.toml

```diff
@@ -86,7 +86,7 @@ docs = [
 ]
 optional = [
     'SimpleITK; sys_platform != "emscripten"',
-    'scikit-learn>=1.2; python_version < "3.14"', # not packaged for Pyodide yet
+    'scikit-learn>=1.2; python_version < "3.14"',
```

Comment:
Assuming you removed this comment because scikit-learn is now available for Python 3.14, why leave in the `< "3.14"` guard?